Vostok.Metrics.System/Host/DiskSpaceInfo.cs

```diff
@@ -0,0 +1,13 @@
+﻿using JetBrains.Annotations;
+
+namespace Vostok.Metrics.System.Host
+{
+    [PublicAPI]
+    public class DiskSpaceInfo
+    {
+        public string Name { get; internal set; }
```

Comment:
What's the point of internal setters? Why should we protect against user creating and filling out an instance of this class?